src/test/java/org/prebid/server/settings/service/JdbcPeriodicRefreshServiceTest.java

```diff
@@ -0,0 +1,176 @@
+package org.prebid.server.settings.service;
+
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.stubbing.Answer;
+import org.prebid.server.VertxTest;
+import org.prebid.server.execution.TimeoutFactory;
+import org.prebid.server.settings.CacheNotificationListener;
+import org.prebid.server.settings.model.StoredDataResult;
+import org.prebid.server.vertx.jdbc.JdbcClient;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class JdbcPeriodicRefreshServiceTest extends VertxTest {
+
+    private static TimeoutFactory timeoutFactory = new TimeoutFactory(Clock.fixed(Instant.now(), ZoneId.systemDefault()));
+
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private CacheNotificationListener cacheNotificationListener;
+
+    @Mock
+    private JdbcClient jdbcClient;
+
+    @Mock
+    private Vertx vertx;
+
+    private Map<String, String> expectedRequests;
```

Comment:
Inline these two vars, it is not the problem duplicate they 2 times but it gives us less cryptic.